Mathlib/Init/Logic.lean

```diff
@@ -78,15 +76,8 @@ attribute [refl] Iff.refl
 attribute [trans] Iff.trans
 attribute [symm] Iff.symm
 
--- This is needed for `calc` to work with `iff`.
-instance : Trans Iff Iff Iff where
```

Comment:
Just flagging this for other reviewers: you are deleting this instance. Do we have a test this works?

Reply:
Yes, it is used in a few places, e.g. Mathlib.Topology.Order.LeftRightNhds and Archive.Imo.Imo1964Q1